packages/manager/modules/vps/src/dashboard/migrate/migrate.controller.js

```diff
@@ -0,0 +1,42 @@
+import set from 'lodash/set';
+import get from 'lodash/get';
+import { MIGRATE_FAQ_LINK } from './migrate.constants';
+
+export default class {
+  /* @ngInject */
+  constructor() {
+    this.MIGRATE_FAQ_LINK = MIGRATE_FAQ_LINK;
+  }
+
+  $onInit() {
+    set(
+      this.newPlan,
+      'configuration',
+      this.parseRangeConfiguration(this.newPlan.planCode),
+    );
+  }
+
+  /* eslint-disable-next-line class-methods-use-this */
+  extractConfigurationFromPlanCode(planCode) {
+    const [cores, memory, storage] = planCode.match(/\d+/g);
+    return [parseInt(cores, 10), parseInt(memory, 10), parseInt(storage, 10)];
+  }
+
+  parseRangeConfiguration(rangeFullName) {
+    const [cores, memory, storage] = this.extractConfigurationFromPlanCode(
+      rangeFullName,
+    );
+
+    return {
+      cpu: { cores },
+      memory: { size: memory },
+      storage: { disks: [{ capacity: storage }] },
+    };
+  }
```

Comment:
This logic is a duplicate of this: 
- https://github.com/ovh/manager/blob/master/packages/manager/modules/vps/src/upscale/upscale.controller.js#L352
- https://github.com/ovh/manager/blob/master/packages/manager/modules/vps/src/upscale/upscale.controller.js#L366

Maybe an helper service could be build for the two use cases?
Also, `/* eslint-disable [...]` should be avoid